### PR TITLE
Update site-impacting-changes to show we've added AVIF support

### PR DESCRIPTION
**Pull request by jasonstephen15**
Rendered article section for review: 
https://review.learn.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes?branch=pr-en-us-3026&tabs=latest#differences-from-the-chromium-schedule-and-high-impact-changes - new row, v121

AB#48573159

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This table lists:
...
 | Ignore modifications to `document.domain` by default | v119 | | The `document.domain` property historically could be set to relax the same-origin policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster). |
 | New TLS server certificate verifier | v109 (unmanaged devices), v111 (managed devices) | | No site compatibility impacts are anticipated.  If you have uncommon TLS server certificate deployments, you should test in v109 to confirm there's no impact. For more information and testing guidance, see [Changes to Microsoft Edge browser TLS server certificate verification](/deployedge/microsoft-edge-security-cert-verification). |
+| Added support for AVIF and AV1 file formats | v121 | | Microsoft Edge now supports the AVIF and AV1 file formats which offer better compression and higher quality images and videos.  Users can enjoy faster loading times and better quality media on websites. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:44]:
```suggestion
| Added support for AVIF and AV1 file formats | v121 | | Microsoft Edge now supports the AVIF and AV1 file formats, which offer better compression and higher quality images and videos.  Users can enjoy faster loading times and better quality media on websites. |
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which
i plan to push this commit

**Review by mikehoffms — APPROVED** [review]
Approved PR per Writer/Editor pass, after pushing commits to:
* Add comma.
* Sort rows decreasing.
* Clarify maintenance comments eg "10 most recent".
* Sentence-case column heading.
* Update date.

**Review by captainbrosset — APPROVED** [review]
